extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -78,7 +78,10 @@ static public function onArticleViewHeader( &$article, &$outputDone, &$useParser
 			}
 
 			// article exists or existed
-			$app->wg->SuppressPageHeader = true;
+			// FIXME wow this is ugly
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)